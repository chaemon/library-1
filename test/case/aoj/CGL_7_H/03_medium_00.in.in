27 10
1 1
1 -6
2 -12
8 -8
4 -9
7 -5
3 -6
7 -2
11 -9
11 -3
4 2
5 4
7 3
12 4
14 6
5 6
9 11
2 8
0 10
-3 3
-3 4
-7 6
-4 8
-8 11
-8 -9
0 -2
-1 0
