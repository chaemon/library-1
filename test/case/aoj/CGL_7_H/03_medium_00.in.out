194.0821733797
